index.js

```diff
@@ -40,7 +40,34 @@ function toggleColor(element) {
   }
 }
 
+const header = document.querySelector("#header")
+header.addEventListener("click", () => {
+	toggleColor(header)
+})
+
 
 /***** Deliverable 2 *****/
 
+const newPlayer = document.querySelector("#new-player-form")
+
+
+newPlayer.addEventListener("sumbit", () => {
+		if (newPlayer.target.elements === "number"); {
+			playerDiv.dataset.number = "number"
+	}	if (newPlayer.target.elements === "name"); {
+			player.name = "name"
+	} if (newPlayer.target.elements === "nickname"); {
+		player.nickname = "nickname"
+	} if (newPlayer.target.elements === "photo"); {
+		player.photo = "photo"
+	}
+
+})
```

Comment:
You've got the right setup here (finding the form element, and then adding a submit event handler to it) - there's a typo though, ("sumbit" instead of "submit"), which will keep the code from running. The next step inside the function here is to use `event.preventDefault()`, otherwise your page will 'refresh' and you won't see your console.log statements. Here's an example solution for this one:

```js
newPlayer.addEventListener("submit", e => {
  e.preventDefault()

  // get values from the form input fields
  const number = e.target.number.value
  const name = e.target.name.value
  const nickname = e.target.nickname.value
  const photo = e.target.photo.value

// create an object, so we can render it using our render function from earlier
  const playerObj = {
    number: number,
    name: name,
    nickname: nickname,
    photo: photo,
    likes: 999
  }

// slap it on the DOM
  renderPlayer(playerObj) 
}
```